ZelBack/src/services/fluxService.js

```diff
@@ -977,6 +977,10 @@ async function getFluxInfo(req, res) {
       throw daemonInfoRes.data;
     }
     info.daemon.info = daemonInfoRes.data;
+    if (info.daemon.info.blocks >= config.syncthingVersionCheckStart) {
+      const versionMinOK = fluxNetworkHelper.minVersionSatisfy(info.flux.syncthingVersion.replace(/[^\d.]/g, ''), '1.27.6');
+      if (!versionMinOK) throw new Error('Syncthing version bellow minimum version allowed 1.27.4, update your machine.');
```

Comment:
The error message version is different that the version we are checking against.

Reply:
nice catch. I have started with 1.27.4 but saw 1.27.5 was already available and... after I have checked they have just released 1.27.6 (despite it wasn't yet available on apt) https://github.com/syncthing/syncthing/releases